packages/wdio-reporter/src/stats/test.ts

```diff
@@ -0,0 +1,83 @@
+import RunnableStats, { RunnableError, Runnable } from './runnable'
+
+export interface Test extends Runnable {
+    cid: string
+    fullTitle: string
+    output: []
+    argument?: Argument
+    retries: number
+    /**
+     * initial test state is pending
+     * the state can change to the following: passed, skipped, failed
+     */
+    state: 'pending' | 'passed' | 'skipped' | 'failed'
+}
+
+interface Argument {
+    rows: [{
+        cells: string[]
+        locations: [{
+            line: number
+            column: number
+        }]
+    }]
+}
+
+/**
+ * TestStats class
+ * captures data on a test.
+ */
+export default class TestStats extends RunnableStats {
+    cid: string
+    title: string
+    fullTitle: string
+    output: {}[]
```

Comment:
Any idea on type?

Reply:
can you please provide some examples of output and I'd suggest a type ;)

Reply:
If I had examples, I wouldn't be asking.

Reply:
You can inspect it with debugger, should answer lot's of questions you have

Reply:
Done